tenzir/functional-test/tests/setup_suite.bash

```diff
@@ -2,29 +2,25 @@ setup_suite() {
   bats_require_minimum_version 1.8.0
   bats_load_library bats-tenzir
 
-  # The default node config is also reasonable for the client commands, ie.
-  # enabling bare mode and disabling plugins etc.
+  export_default_paths
   export_default_node_config
+  
+  export INPUTSDIR="${BATS_TENZIR_INPUTSDIR}"
+  export QUERYDIR="${BATS_TENZIR_QUERIESDIR}"
+  export MISCDIR="${BATS_TENZIR_MISCDIR}"
 }
 
-SCRIPT_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)
-export BATS_LIB_PATH=${BATS_LIB_PATH:+${BATS_LIB_PATH}:}${SCRIPT_DIR}/../lib
-
-# Normalize the environment unless `BATS_TENZIR_KEEP_ENVIRONMENT` is set.
-if [[ ! -n ${BATS_TENZIR_KEEP_ENVIRONMENT} ]]; then
-  unset $(printenv | grep -o '^TENZIR[^=]*' | paste -s -)
-fi
+teardown_suite() {
+  : # Nothing to do
+}
 
-if ! which tenzir-node; then
+if ! which tenzir; then
   echo "tenzir binaries must be on $PATH"
   return 1
 fi
 
-# TODO: Should the datadir definitions move into the bats-tenzir library,
-# so that files in there automatically available for plugins integration tests?
-BATS_TEST_DATADIR="$(realpath $(dirname ${BATS_TEST_DIRNAME}))"
-export BATS_TENZIR_DATADIR="${BATS_TEST_DATADIR%%/functional-test/*}/data"
-
-export INPUTSDIR="${BATS_TENZIR_DATADIR}/inputs"
-export QUERYDIR="${BATS_TENZIR_DATADIR}/queries"
-export MISCDIR="${BATS_TENZIR_DATADIR}/misc"
+# Try to load the `bats-tenzir` library either relative to
+# the current directory (for in-tree builds) or relative to the
+# tenzir binary (for builds against an installed tenzir)
+TENZIR_DIR="$(realpath "$(dirname "$(command -v tenzir)")")"
+export BATS_LIB_PATH=${BATS_LIB_PATH:+${BATS_LIB_PATH}:}${TENZIR_DIR}/../share/tenzir/functional-test/lib:${BATS_TEST_DIRNAME}/../../../../../tenzir/functional-test/lib
```

Comment:
Can you add a comment here explaining why we step out 5 times?

Reply:
When I commented this I did not know that we have this copied over for every plugin. Let's revisit this later.

Reply:
There is one in line 24, but I guess i should reword it to be more obvious :D